lib/target/users.ex

```diff
@@ -8,23 +8,19 @@ defmodule Target.Users do
   alias Target.Repo
   alias Target.Users.User
 
-  def list_users do
-    Repo.all(User)
-  end
-
   def get_by_email(email) do
     Repo.get_by(User, email: email)
   end
 
-  def current_email_unconfirmed?(%{
+  def current_email_confirmed?(%User{
```

Comment:
You changed the name but not the implementation? 🤔

Reply:
About that... after double checks I realized the name was mistaken👀